dask/dataframe/methods.py

```diff
@@ -324,8 +325,24 @@ def size(x):
     return x.size
 
 
-def values(df):
-    return df.values
+def to_numpy(df, _dtype, na_value):
+    kwargs = {"dtype": _dtype, "copy": False}
+
+    if PANDAS_GT_110:
+        # callers use the "__no_default__" string instead of `pd_no_default`,
+        # since `pd_no_default` isn't consistently pickleable or tokenizeable
+        # https://github.com/pandas-dev/pandas/issues/40397
+        from .core import no_default
+        from pandas.api.extensions import no_default as pd_no_default
+
+        try:
+            if na_value == no_default:  # `==` not `is`, since it could be pickled
+                na_value = pd_no_default
+        except Exception:
```

Comment:
Is there a more specific exception we can catch here? When I do `bool(pd.NA == dd.core.no_default)` I get `TypeError: boolean value of NA is ambiguous`